sdk/iot/inc/az_iot_provisioning_sas.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <az_span.h>
+#include <az_result.h>
+#include <time.h>

```

Comment:
[Link to Jeffrey's comment about using int for time](https://github.com/Azure/azure-sdk-for-c/pull/302#discussion_r365350923)

Reply:
[Link to http using `uint16_t` for timing](https://github.com/Azure/azure-sdk-for-c/blob/25f8a0228e5f250c02e389f19d88c064c93959c1/sdk/core/core/inc/az_http.h#L171)

Reply:
The links are for offset timing not for RTC. We need a very specific formatted RTC value (UnixTime) and that is obtainable through a well-defined and C99-required interface in time.h if a RTC / NTP system is available.
The lib will need to understand the system epoch then calculate the offset from the one expected by IoT Hub/DPS.

Reply:
We took a decision to use a uint32_t instead of relying on time.h. We expect that the value represents the correct Unix epoch time.